### DISCUSSION TITLE
How to migrate to V1.0.0 - CLI not working?

### DISCUSSION THREAD

**Discussion by cjlotz** [cake-build/cake#3100] [Q&A] [answered]
Hello

I read the docs around the [CLI updates](https://cakebuild.net/docs/getting-started/upgrade#cake-cli-updates) for V1.0, but still not getting the builds to fire on Mac.  Here's the command line I'm using:

`./build.sh --target=Rebuild --configuration=Debug`

and the issue I'm seeing

<img width="1109" alt="Screenshot 2021-02-08 at 10 36 40" src="https://user-images.githubusercontent.com/1300847/107195098-93a88480-69f9-11eb-847a-353a6f0e8d94.png">

**Comment by gep13** [comment, marked as answer]
@cjlotz With the switch to using a new command line parser, all input arguments must now use `-` when passing a single character argument, and `--` when using an argument that has multiple characters.

As such, if you switch to using: `--verbosity` rather than `-verbosity`, things should hopefully start working for you.

**Reply by cjlotz** [reply]
@gep13 Thanks for the reply, but I'm not specifying verbosity and when I specify it I still get an error.  What am I missing?

**Reply by gep13** [reply]
Check inside the build.sh file that you are using.

**Reply by cjlotz** [reply]
Of course, forgot that the build.sh is also checked into source control

**Reply by pascalberger** [reply]
We have up to date templates for bootstrappers available here: https://github.com/cake-build/resources
